cpu/fe310/Kconfig

```
@@ -64,3 +64,5 @@ config CPU_MODEL
 
 config CPU
     default "fe310" if CPU_FAM_FE310
+
+orsource "Kconfig.clk"
```

I don't think we need an optional include here, the file is always there:
```suggestion
rsource "Kconfig.clk"
```